### ISSUE TITLE
Support for tmux-battery and tmux-cpu

### ISSUE THREAD

**Issue by Laleee** [arcticicestudio/nord-tmux#22]
Hi,

I wanted to use tmux-battery and tmx-cpu plugins with this theme but icons seem to be behind status content. Can you add support for those?

Loved the theme!

**Comment by zhuxiaoxi** [comment]
Yes, I have encountered the same problem. The status bar is overwritten after using the theme.

**Comment by arcticicestudio** [comment]
By default, this theme provides status bar content (including the powerline-styled background arrows).
The status content can be disabled by setting the `@nord_tmux_show_status_content` to `0`:

```sh
set -g @nord_tmux_show_status_content "0"
```

**Comment by gusgamble** [comment]
For anyone looking, there is a way to modify what is already there. Note that this requires tmux-cpu to be installed. 
This is what I was able to achive: 
<img width="326" alt="Screen Shot 2019-11-18 at 3 30 07 AM" src="https://user-images.githubusercontent.com/32199979/69036471-f3a9aa00-09b3-11ea-820c-4578db407ac6.png">
and it does update with real time readings

**Steps:**
1.  make a backup of the existing config just in case`cd ~/.tmux/plugins/nord-tmux/source/ && cp nord-status-content.conf nord-status-content.conf.bak` *note that the path you have might be different

2.   replace the `status-right` formatting with:
```tmux
set -g status-right "#{prefix_highlight}#[fg=brightblack,bg=black,nobold,noitalics,nounderscore]#[fg=white,bg=brightblack] CPU: #{cpu_icon} #{cpu_percentage} #[fg=white,bg=brightblack,nobold,noitalics,nounderscore]#[fg=white,bg=brightblack] %H:%M #[fg=cyan,bg=brightblack,nobold,noitalics,nounderscore]#[fg=black,bg=cyan,bold] #H "
```

I decided to remove the date from the Nord bar and replace that, but it should be the same procedure to swap others too.

Anyway, that's just my 5-second hack for it. This is all untested, and will inevitably be lost during any updating so...
